/* 
 * File:   Vision.hpp
 * Author: attilio
 *
 * Created on February 15, 2013, 6:06 PM
 */

#ifndef RIGIDFORM_HPP
#define	RIGIDFORM_HPP

//#define __GXX_EXPERIMENTAL_CXX0X__
#include <thread>
#include <mutex>
#include <cstdlib>
#include <cstdio>
#include <sstream>
#include <iostream>
#include <cctype>
#include <algorithm>
#include <string>
#include <boost/algorithm/string.hpp>
#include <ros/ros.h>
#include <saetta_msgs/cmd_vel.h>
#include <math.h>
#include <tf/transform_listener.h>
namespace rigidform
{


};
#endif	/* VISION_HPP */

